pyproject.toml

```diff
@@ -84,6 +84,7 @@ tests = [
 lint = [
   "pre-commit==4.2.0",
   "pre-commit-uv==4.1.4",
+  "ruff>=0.11.9"
```

Comment:
why is this needed?

Reply:
Sometimes I run `ruff format .` to do a quick cleanup of a file without having to wait for the whole pre-commit cycle, i.e. running `pre-commit run --all-files`. I don't understand how it works but I needed this to be able to run ruff directly

Reply:
You can run `pre-commit run ruff` being `ruff` any of the `id`s as in:

https://github.com/mozilla-ai/any-agent/blob/a9ee6f83eab094bcdb63e33cff88f776fb7dbe2f/.pre-commit-config.yaml#L23

Reply:
Hmm. When I run that I get 
```
(any-agent) *[main][~/scm/any-agent]$ pre-commit run ruff-format  
[WARNING] Unstaged files detected.
[INFO] Stashing unstaged files to /Users/nbrake/.cache/pre-commit/patch1747250783-25619.
ruff-format..........................................(no files to check)Skipped
```
so it stashes my unstaged changes, runs format, and then pops my changes back from the stash lol...

So that doesn't work

Reply:
I see. 

if you want to run on unstaged, you could:

```
pre-commit run ruff-format --files $(git ls-files -m)
```

not ideal (maybe you can alias that) but I would prefer to not have the separate ruff version dep as we will, most likely, start to encounter version discrepancies

Reply:
Nice, that works 🚀 . I'll undo adding the ruff dependency 👍